src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -1051,9 +1068,9 @@ private bool FileValidationPassed(List<Answer> answers, Page page, List<Validati
         }
 
         [HttpPost]
-        public async Task<IActionResult> Submit(Guid applicationId, int sequenceId)
+        public async Task<IActionResult> Submit(Guid applicationId)
```

Comment:
This controller action has been removed in future branches, the check should be done in SubmitApplication / ConfirmSubmitApplication if not done already

Reply:
this will have to be done retrospectively

Reply:
included check in ConfirmSubmitApplication